demo-linux/demo.cpp

```diff
@@ -249,6 +247,10 @@ void parseArgs(int argc, char** argv)
             else
                 usage();
         }
+        else if (!strcmp(argv[i], "--write-eeprom"))
+        {
+			writeToEEPROM();
```

Comment:
I'm going to have to ask, how did you test --write-eeprom and --integration-time-ms?  I have trouble believing they work, because you're calling writeToEEPROM() and wp_set_integration_time_ms() from inside parseArgs().

I can see in main() that parseArgs() is called BEFORE init(), which is what actually establishes the connection to the spectrometer.  So this seems unlikely to work, and shouldn't have passed testing.  

Please ensure new code is tested before code review.

Reply:
Note to preceding: it looks like I committed --integration-time-ms

Reply:
My implementation was very different though: https://github.com/WasatchPhotonics/Wasatch.VCPP/commit/50ecce019b0644495872fb7975a870f0a687e0b9

I'm not sure what happened here.